composer.json

```diff
@@ -14,20 +14,25 @@
         "sensio/framework-extra-bundle": "^5.1",
         "sensiolabs/security-checker": "^5.0",
         "symfony/asset": "*",
+        "symfony/console": "*",
         "symfony/expression-language": "*",
         "symfony/flex": "^1.1",
         "symfony/form": "*",
         "symfony/framework-bundle": "*",
+        "symfony/intl": "*",
         "symfony/monolog-bundle": "^3.1",
+        "symfony/options-resolver": "*",
```

Comment:
About `OptionsResolver` it's a hard dep for `symfony/form` (which is already required), so there is no effect.